decisions/inexact-rest.md

```diff
@@ -0,0 +1,56 @@
+# Inexact ...rest and TypeScript
+
+## TL;DR
+
+If using `...rest` parameter with the intention to allow any arbitrary prop to be passed then type using:
+
+```tsx
+type Props = {
+  namedPropOne: string;
+  namedPropTwo: boolean;
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
+};
+```
+
```

Comment:
We've got a lot of 
```
// $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
```

in the codebase right now.

While adding the type as above means no suppression is necessary for TypeScript I do think it is worth keeping the inline comment, for the reasons given in the doc.

WDYT?